courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>do you need to map the "objectives" to individual parts of the code as we did for other labs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>